### DISCUSSION TITLE
Will there be any updates in regards to the Veracrypt issue?

### DISCUSSION THREAD

**Discussion by RedRain77** [Difegue/LANraragi#494] [Q&A] [answered]
<!-- hol up senpai have you checked out the GH Discussions first ? 
Your issue/idea might've already been reported by someone else! 🐱‍👤
https://github.com/Difegue/LANraragi/discussions
->

**LRR Version and OS**
LANraragi v.0.7.9, Windows 10 19041.685

**Bug Details**
Is there any update on the bug reported [here](https://github.com/Difegue/LANraragi/issues/182)?

**Comment by RedRain77** [comment]
Sorry, My description seemed to disappear.

**LRR Version and OS**
LANraragi v.0.7.9, Windows 10 19041.685

**Bug Details**
is there any update in regards to the bug reported [here](https://github.com/Difegue/LANraragi/issues/182), I'm currently using a veracrypt volume for the bulk of manga/doujinshi.

**Comment by Difegue** [comment, marked as answer]
VeraCrypt volumes can supposedly work if you use WSL2(can be toggled in the settings of the Windows app), according to the comment here: https://github.com/Microsoft/WSL/issues/530#issuecomment-866342690 .   

We're hitting niche-within-niche usages here, so I don't really plan on adding explicit support for Veracrypt volumes to the Windows helper app at the moment.
